workflows/RNA-seq/RNA-seq

```diff
@@ -283,7 +290,22 @@ def main():
     ## save to configs.yaml in outdir
     cf.write_configfile(os.path.join(args.outdir,'config.yaml'),config)
 
-    snakemake_module_load = "module load snakemake/3.12.0 slurm &&".split()
+    ## cluster config
+    cluster_config = cf.load_configfile(os.path.join(this_script_dir,"../../shared/","cluster.yaml"),False)
+    cluster_config = cf.merge_dicts(cf.load_configfile(os.path.join(this_script_dir,"cluster.yaml"),False), cluster_config)
+    
+    if args.cluster_configfile:
+        user_cluster_config = cf.load_configfile(args.cluster_configfile,False)
+        cluster_config = cf.merge_dicts(cluster_config, user_cluster_config) # 2) form user_config.yaml
+    cf.write_configfile(os.path.join(args.outdir,'cluster_config.yaml'),cluster_config)
+    
+    snakemake_module_load = "module load snakemake/3.12.0 "
```

Comment:
If we want to publish this then the `module load` command should be put in a yaml. The code would then need to check if this is blank and not append `&&` as approriate.

Reply:
you mean including the `module load` part? ie. to be independent of "module" in the  wrapper and  put whatever is necessary to get the right snakemake path in a configurable yaml entry!?